components/brave_shields/content/browser/ad_block_service.cc

```diff
@@ -209,6 +208,41 @@ base::Value::Dict AdBlockService::UrlCosmeticResources(
         }
       }
     }
+
+    base::Value::List* procedural_actions =
+        resources.FindList("procedural_actions");
```

Comment:
seems like `"procedural_actions"` should maybe be a const? It's used a a string literal in a lot of places and that's error prone

Reply:
updated